platform/keysystem/keyserver/keyserver.go

```diff
@@ -15,7 +16,12 @@ func main() {
 	_, onstop, err := keyapi.Run(":20557", logger)
 	if err != nil {
 		logger.Fatal(err)
+	}
+	err = exec.Command("systemd-notify", "--ready").Run()
+	if err != nil {
+		logger.Fatal("failed to notify systemd of readiness: %v\n", err)
 	} else {
```

Comment:
mind removing the other one too, while you're at it?

Reply:
extraneous `else` clause removed in 6ae8ae9bc86c6781a41dfb28b72d9c6ad29d97d4.